gui/src/ts/components/node/NodePredicateList.tsx

```diff
@@ -26,31 +26,32 @@ const useStyles = makeStyles(() =>
         padding: "16px",
       },
       "&:last-child > *": {
-        paddingBottom: "24px"
-      }
+        paddingBottom: "24px",
```

Comment:
Use theme.spacing. Default scaling factor is 8px, so this would be theme.spacing(3).

Reply:
this is a bit of a hack.  These specific values are overriding the padding values of the material-ui element, which don't seem to scale with our theme